internal/linker/linker.go

```diff
@@ -64,23 +67,16 @@ func loadLinkerPatches() (string, map[string]string, error) {
 			if file.IsNew || file.IsDelete || file.IsCopy || file.IsRename {
 				panic("only modification patch is supported")
 			}
-			patches[file.OldName] = string(patchBytes)
+			modFiles = append(modFiles, file.OldName)
```

Comment:
you might want to deduplicate this list - or use a `map[string]bool` as a string set, since I don't think we really care about the order.